src/commands/logs/build.mts

```diff
@@ -0,0 +1,92 @@
+import { OptionValues } from 'commander'
```

Comment:
Given this is only being used for the type you can change this import to:

```suggestion
import type { OptionValues } from 'commander'
```

Reply:
Cool - we should make this a ESLint rule if we're enforcing this

Reply:
Agree with eslint rule